provider/tests/test_mgmt_cmd.py

```diff
@@ -8,6 +8,7 @@
 from django.core.management.base import CommandError
 from provider.management.commands.delete_expired_grant_tokens import Command
 from provider.oauth2.models import Grant, AccessToken, RefreshToken
+from six.moves import range
```

Comment:
It would be better to just use the built-in range in this file, no?  And might as well change `range(0, 10)` to `range(10)`.

Reply:
Ned, you're right, thanks.
I just updated the PR with proposed changes.